curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -6,8 +6,7 @@ dashedName: step-12
 ---
 
 # --description--
-
-Turn the words `cat photos` located inside `p` element into a link using the same value for the `href` attribute as the link below the `p` element. The `p` element should show the same text in the browser, but the words `cat photos` should now be a link. Make sure to remove the `a` element with the text `cat photos` on the line below the `p` element.
+Turn the words `cat photos` located inside the `p` element into a link by wrapping them in an anchor (`a`) element. Use the same link from the previous challenge (`https://freecatphotoapp.com`) as the value for the `href` attribute. The `p` element will have the same text in the preview, but the words `cat photos` will be a link. Be sure to remove the `a` element with the text `cat photos` on the line below the `p` element. 
```

Comment:
We specifically went away from using () around code blocks due to confusion. We probably don't want to add them back here.

Reply:
Note - I put some comments into the original issue. The issue was not open for contribution yet since we have not actually decided upon a way forward yet.